_topic_map.yml

```diff
@@ -2113,7 +2113,7 @@ Topics:
   Topics:
   - Name: Installing Jaeger
     File: rhbjaeger-installation
-  - Name: Deploying Jaeger
+  - Name: Configuring Jaeger
```

Comment:
Is it configuring Jaeger or Configuring and Deploying Jaeger. Is the difference between topicmap.yml and the h1 in the title of the assembly on purpose?

Reply:
One of my developers was asking where he could find the configuration information in the docs.  I realized that customer are probably going to be thinking the same thing, where's the configuration.  I would have put both "configuration and deploying" in the navigation, but I was trying to avoid having the heading wrap.  But I kept both in the heading for searchability.